src/main/java/ExpiredTokenJob.java

```diff
@@ -0,0 +1,53 @@
+import fr.maif.socleapi.croupier.model.TokenInfos;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.mail.MailException;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+public class ExpiredTokenJob {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ExpiredTokenJob.class);
+
+    @Autowired
+    private TokenService tokenService;
+
+    @Autowired
```

Comment:
Unused field 'helloBlabla'. Consider removing it to avoid confusion and reduce clutter.